tests.py

```diff
@@ -92,6 +123,9 @@ def test_configfile_loading(self, mock_user_input):
             " postwhite spamassassin uwsgi",
             out.getvalue()
         )
+        self.assertNotIn("It seems that your config file is outdated.",
+                         out.getvalue()
+        )
```

Comment:
It could be useful to also validate the detection of outdated files. What do you think?

Reply:
It is tested above (around line 60). You were thinking about another approach for the test?